### PR TITLE
feat: launch the exports for all organizations that have the checkbox…

### LINKED ISSUE #6558: Activate automated daily exports of product data and photos to the public database
### Is your feature request related to a problem? Please describe.

We have nightly imports of data from Equadis into the producers platform. And we have a nightly cron to export data from the producers platform to the public platform. This export is currently done for each organization, and the corresponding organizations have to be listed in scripts/export_producers_platform_data_to_public_database.sh

That means that new organizations that send us data from Equadis don't see their data on the public platform until we add them to the script.

### Describe the solution you'd like

We already have a checkbox "Activate automated daily exports of product data and photos to the public database." in the admin fields of each organization.

- Check that the checkbox is checked when orgs are created by Equadis and Agena3000 imports.
- Instead of listing each organization in scripts/export_producers_platform_data_to_public_database.sh , launch the exports for all organizations that have the checkbox checked.

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### Number of products impacted

_No response_

### Time per product

_No response_

### PR DESCRIPTION
**Pull request by 4nt0ineB**
… checked

### What

Instead of listing each organization in scripts/export_producers_platform_data_to_public_database.sh , launch the exports for all organizations that have the checkbox checked.

### Related issue(s) and discussion

- Fixes #6558

### CODE REVIEW

File: scripts/export_producers_platform_data_to_public_database.sh [hunk 1]
```diff
@@ -4,28 +4,51 @@
...
-org-nestle-waters
-org-kambly
-org-kambly-france
-org-saint-hubert
-org-d-aucy
-org-lea-nature
-org-auchan-apaw
-org-les-mousquetaires
-)
+filter_organizations_that_have_automated_export() {
```
Comment (alexgarel) [inline scripts/export_producers_platform_data_to_public_database.sh:7]: Shan't we leverage having orgs in MongoDB now and just make a mongoDB request (you can even user mongo client) to get all orgs with activate_automated_daily_export_to_public_platform on ?

cc: @stephanegigandet

Comment (4nt0ineB) [reply scripts/export_producers_platform_data_to_public_database.sh:7]: Oh yes I forgot, sure we can do that.

File: scripts/export_producers_platform_data_to_public_database.sh [hunk 2]
```diff
@@ -4,30 +4,48 @@
...
-
-for producer in ${PRODUCERS[@]}
+filter_organizations_that_have_automated_export() {
+    perl -e '
+        use strict;
+        use warnings;
+        use ProductOpener::Data qw/get_orgs_collection/;
+
+        my %producers = map { $_,1 } qw(
+            org-barilla-france-sa
```
Comment (stephanegigandet) [inline scripts/export_producers_platform_data_to_public_database.sh:14]: We probably should remove the hardcoded values, and ensure all those orgs have the checkbox checked

**Review by stephanegigandet — APPROVED** [review]
Looks good to me

File: scripts/export_producers_platform_data_to_public_database.sh (hunk 2 above)
Comment (4nt0ineB) [reply scripts/export_producers_platform_data_to_public_database.sh]: Done. But I'm not an admin so I can't see the checkbox in the platform.

**Pull request merged**